src/Api/GitHub/Updater.php

```diff
@@ -190,36 +190,34 @@ protected function httpRequestSslVerify(array $args, string $url): array
 
     /**
      * Hook into the plugin update check and connect to GitHub
-     * @param object $value the plugin data transient
+     * @param mixed $value the plugin data transient
      * @return object $transient updated plugin data transient
      */
     protected function apiCheck($value)
     {
-        if (!isset($value) ||
-            !\is_object($value) ||
-            (empty($value->response) && !\is_array($value->response)) ||
-            !empty($value->checked)
```

Comment:
- a function's parameter is always set - although it can be `null`
- `&&` was impossible
- `checked` checking was contradicting the next `if`